test/src/sdk-harness/test_projects/reentrancy/mod.rs

```diff
@@ -29,7 +29,7 @@ async fn can_detect_reentrancy() {
 }
 
 #[tokio::test]
-#[should_panic(expected = "Revert(0)")]
+#[should_panic(expected = "Revert(18446744073709486084)")]
```

Comment:
As mentioned [here](https://github.com/FuelLabs/sway/issues/3950#issuecomment-1411333835), ideally we'd have fuel crate with some consts that make this a bit more readable, e.g.

```suggestion
#[should_panic(expected = "Revert(fuels_abi_types::FAILED_ASSERT_SIGNAL)")]
```
but this can be handled in follow-up.

Reply:
Even more ideally they wouldn't have to be sync'd manually between the Sway lib and Rust crates.